istio.VERSION

```diff
@@ -4,11 +4,12 @@ export CA_HUB="gcr.io/istio-testing"
 export CA_TAG="b7b8c10bbb6ca471e9c8aee2b3889f0ccbf01c3c"
 export MIXER_HUB="gcr.io/istio-testing"
 export MIXER_TAG="4e107a2411e1d58e88fd9ae4d32a139751e87955"
-export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/3ac357f84576f881d9f57ed76e4e14a052ac5333/artifacts/istioctl"
 export PILOT_HUB="gcr.io/istio-testing"
 export PILOT_TAG="3ac357f84576f881d9f57ed76e4e14a052ac5333"
+export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/3ac357f84576f881d9f57ed76e4e14a052ac5333/artifacts/istioctl"
```

Comment:
moved after PILOT_TAG so we can eventually change it to
export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/$PILOT_TAG/artifacts/istioctl"